packages/@lightningjs/ui-components/src/components/Badge/Badge.mdx

```diff
@@ -1,23 +1,8 @@
-﻿<!--
```

Comment:
Temporarily removing licenses in mdx files until the linter is fixed. MDX2 doesn't use HTML comments `<! ---`  must switch to JS comments `{/* comment here */}` The JS comment works if there isn't multiple lines, but will change to: ( when hitting save and linter corrects, I believe it may have to to do with the TS linter overrides)
```
{/_ this is a comment add more to this comment
add another line
add a third line
_/}
```

Reply:
This has been fixed now. Commented license has been added back to all .mdx files. Please note the updated syntax for MDX2 files and the ignore linter